style.css

```diff
@@ -0,0 +1,4 @@
+.header {
+  color: #291b1b;
+  background: #fff;
+}
```

Comment:
- [ ] [REQUIRED] Good job so far! Kindly your style is not working, try to give style to your `h1` tag 👍🏼 

![image](https://user-images.githubusercontent.com/63915024/226962618-e6acbeee-a172-4f7c-878f-2cc26784ae21.png)